docs/wrapper.rst

```diff
@@ -1,6 +1,12 @@
-===============
```

Comment:
It seems like you reverted the changes from https://github.com/ROCm/composable_kernel/pull/1098 in this file. There was probably a merge conflict between your branch and the changes pushed in https://github.com/ROCm/composable_kernel/pull/1098.

Reply:
I have updated wrapper.rst with your changes, and merged my changes. Thanks for catching the problem.

Reply:
That's great, thanks!